src/OctopusPuppet.OctopusProvider/OctopusDeploymentPlanner.cs

```diff
@@ -312,7 +322,7 @@ public EnvironmentDeploymentPlans GetEnvironmentMirrorDeploymentPlans(string env
 
             foreach (var dashboardProjectResource in dashboard.Projects)
             {
-                if (_repository.Projects.Get(dashboardProjectResource.Id).IsDisabled)
+                if (_cachedProjects.First(x => x.Id == dashboardProjectResource.Id).IsDisabled)
```

Comment:
if you assume that there should only be one in _cachedProjects matching dashboardProjectResource.Id then First() should be Single()

Reply:
done. thanks.